webapp/src/components/github_issue_selector/index.js

```diff
@@ -0,0 +1,7 @@
+import {connect} from 'react-redux';
+
+import GithubIssueSelector from './github_issue_selector';
+
+const mapStateToProps = () => ({});
+
+export default connect(mapStateToProps, null, null, {withRef: true})(GithubIssueSelector);
```

Comment:
Is this file necessary if we are not injecting any state or actions into the component?

Reply:
removed